src/connectionHandler.ts

```diff
@@ -72,12 +72,19 @@ const initCmdObserver = async (
 
   const env = initEnvironment()
 
-  env.sendMessage.subscribe(msg => {
-    const embed = new MessageEmbed().setDescription(msg)
-    message.channel.send({
-      embeds: [embed]
-    })
-  })
+  env.sendMessage.subscribe(async msg => sendMessage({ msg, message })
+    .then(sentMsg => {
+      if (sentMsg.embeds[0].title?.startsWith('Queue')) {
+        env.reprintQueueOnReaction.next(sentMsg)
+      }
+    }))
+
+  env.editMessage.subscribe(async editedMessage => editMessage(editedMessage)
```

Comment:
Kan använda async-await här om du föredrar det
```suggestion
  env.editMessage.subscribe(async editedMessage => {
    const editedMsg = await editMessage(editedMessage)

    if (editedMsg.embeds[0].title?.startsWith('Queue')) {
      env.reprintQueueOnReaction.next(editedMsg)
    }
  })
```

Reply:
Samma här som ovan